README.md

```diff
@@ -97,6 +97,7 @@ parameters are used:
 *  **binddn** ... distinguished name (DN) to bind when reading the user’s entry (default is to bind
                   anonymously).
 *  **bindpw** ... credentials to bind with when reading the user’s entry (default is none).
+*  **ssl** ... does the LDAP server require starttls
```

Comment:
This description is misleading, the `ssl` parameter is not boolean, but `start_tls`, `on`, `off` (not sure about this), or nothing (I guess that it means off). The default value is nothing/off.